tools/nl/eval/fetch_explore_queries.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+from absl import app
+from urllib.parse import quote
+import requests
+import pandas as pd
+
+EXPLORE_CONFIG_RELATIVE_PATH = '../../../static/js/apps/explore_landing/topics.json'
+NODEJS_URL = 'https://bard.datacommons.org/nodejs/query'
+API_KEY = os.environ.get('NODEJS_API_KEY')
+
+
+def call_dc(query):
+  full_url = f'{NODEJS_URL}?q={quote(query)}&apikey={API_KEY}'
+  response = requests.get(full_url).json()
+  if 'debug' not in response:
+    print("!!!!! no data")
+    return []
+  debug = response['debug']['debug']
+
+  single_sv_best_score = debug['sv_matching']['CosineScore'][0]
+  use_single_sv = True
+  multi_sv_candidate = None
+  if 'MultiSV' in debug['sv_matching']:
+    for candidate in debug['sv_matching']['MultiSV']['Candidates']:
+      if candidate['DelimBased'] and len(candidate['Parts']) == 2:
+        if candidate['AggCosineScore'] + 0.05 > single_sv_best_score:
```

Comment:
curious what the `+ 0.05` is for, maybe leave a comment?

Reply:
This is to match nl logic. Added a comment.